src/common/AreaHeading.tsx

```diff
@@ -4,18 +4,19 @@
  * SPDX-License-Identifier: MIT
  */
 import { Box, Text } from "@chakra-ui/layout";
+import { ReactNode } from "react";
 
 interface AreaHeadingProps {
   name: string;
-  description: string;
+  description: string | ReactNode;
```

Comment:
I think this can just be ReactNode.

Reply:
[`9a2a136`](https://github.com/microbit-foundation/python-editor-next/pull/807/commits/9a2a13661160c3d4201c90b1113b0dbf50cf5934).